### PR TITLE
feat: Applicant landing page adjustments (HL-1033)

### PR DESCRIPTION
**Pull request by EmiliaMakelaVincit**
## Description :sparkles:

Adjusts the landing page of the benefit applicant site to include a reminder about prerequisites for applying, as well as the style and text content of its introductory heading section.

### CODE REVIEW

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2709.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2709.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2709.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

File: frontend/benefit/applicant/src/components/mainIngress/MainIngress.tsx [hunk 1]
```diff
@@ -24,36 +25,38 @@ const MainIngress: React.FC = () => {
...
+        <Container>
+          <$Heading>{t('common:mainIngress.heading')}</$Heading>
+          {notificationItems}
+          <$TextContainer>
+            <$Description>
+              {t('common:mainIngress.description1')}
+              {/* TODO: uncomment once having link to redirect to more info url,
             handleMoreInfoClick is from useMainIngress */}
-            {/* <$Link onClick={handleMoreInfoClick}>
+              {/* <$Link onClick={handleMoreInfoClick}>
```
Comment (sirtawast) [inline frontend/benefit/applicant/src/components/mainIngress/MainIngress.tsx:39]: This is ok for me and I see the point in constant structural improvement.

In general, we've previously discussed with @rikuke and @mjturt that commented code should be left out. An example of this is that there's plenty of badly aged TODO's in the code (especially in Django side) that the current team has no idea of.

**Comment by EmiliaMakelaVincit** [comment]
Fair enough; I've now updated the prerequisite box styles to follow the design more closely.

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr2709.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr2709.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**